mysql-test/suite/innodb/r/foreign_null,INPLACE.rdiff

```diff
@@ -0,0 +1,20 @@
+--- foreign_null.result
++++ foreign_null,INPLACE.result
+@@ -45,7 +45,7 @@
+ CREATE TABLE t1(f1 INT, f2 INT NOT NULL, PRIMARY KEY(f1), KEY(f2))ENGINE=InnoDB;
+ CREATE TABLE t2(f1 INT NOT NULL, FOREIGN KEY(f1) REFERENCES t1(f2) ON UPDATE CASCADE)ENGINE=InnoDB;
+ ALTER TABLE t1 MODIFY COLUMN f2 INT;
+-ERROR HY000: Cannot change column 'f2': used in a foreign key constraint 't2_ibfk_1' of table 'test.t2'
++ERROR HY000: Cannot change column 'f2': used in a foreign key constraint 't2_ibfk_1' of table '`test`.`t2`'
```

Comment:
I don’t think that this file is allowed to exist. The error message output needs to be identical in both cases. See 024a18dbcb7f9c44094db03daa013fd8ba3c58b7 how the table name can be reported properly.